woocommerce/woocommerce-gutenberg-products-block
# [Store Customization] Fix "Discount Banner" pattern

## What

Fixes the white space on the right of the pattern by using groups instead of columns.

## Why

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11235


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Add the `Discount Banner` pattern.
3. Check that is centered on the page (like the "after" banner in the screenshot 👇).
4. Check it works well on mobile as well.


## Screenshots or screencast


![CleanShot 2023-10-18 at 14 36 38@2x](https://github.com/woocommerce/woocommerce-blocks/assets/186112/00dfd99e-e997-49a3-b1c5-5397ccdd1b0f)


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Improve the "Discount banner" pattern.
